01_P/P_2_0_02/sketch.js

```diff
@@ -18,22 +18,24 @@
 
 /**
  * drawing with a changing shape by draging the mouse.
- *   
+ *
  * MOUSE
  * position x          : length
  * position y          : thickness and number of lines
  * drag                : draw
- * 
+ *
  * KEYS
  * del, backspace      : erase
  * s                   : save png
  */
+'use strict';
 
 function setup() {
   createCanvas(720, 720);
-  smooth();
```

Comment:
Removed smooth() since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).